integration/grafana/README.md

```diff
@@ -23,3 +23,83 @@ is the `geopmexporter(1)`
 file - `GEOPM_Report.grafana.dashboard.json`, in this directory, 
 relies on `geopmexporter(1)` for its data source.
 
+
+## Monitoring PBS jobs
+
+The `geopmexporter(1)` is typically deployed system wide with either a systemd
+service or as a kubernetes service.  This documentation is for an alternative
+`geopmexporter(1)` deployment scenario where an unprivileged user of a PBS
+managed system would like to monitor the compute resources granted to the user
+only while they are allocated.  This methodology should not be applied for
+system-wide monitoring by an administrator, but rather for job monitoring by an
+end user of a PBS system where the geopmexporter is not deployed system-wide.
+
+A script is provided called `pbs_prometheus_launch.sh` that can be used to
+monitor jobs submitted to a PBS batch system.  The script runs the prometheus
+and grafana server on the head node of a system, and will connect to
+`geopmexporter(1)` clients running on the compute nodes of a user's job.
+
+### Requirements
+
+A user installation of grafana and prometheus is required.  We recommend
+following the installation instructions
+[here](https://grafana.com/grafana/download?platform=linux) to `wget` the
+grafana archive for Linux.  The expanded archive is the `GRAFANA_DIR` provided
+to the `pbs_prometheus_launch.sh` script CLI.  To install prometheus download
+the archive for the latest stable Linux release from
+[here](https://prometheus.io/download/).  The expanded archive is the
+`PROMETHEUS_DIR` provided to the `pbs_prometheus_launch.sh` script CLI.
+
+Until the `geopmexporter(1)` is available in a stable release, the user must
+build and install a development snapshot of GEOPM.
+
+```
+
+    # Install latest development snapshot of GEOPM
+    GEOPM_DIR=/mnt/scratch/$USER/geopm-build
+    wget https://raw.githubusercontent.com/geopm/geopm/refs/heads/dev/geopmdpy/install_user.sh
+    chmod a+x install_user.sh
+    ./install_user.sh --prefix=$GEOPM_DIR --enable-levelzero
+
+```
```

Comment:
```suggestion
```bash
# Install latest development snapshot of GEOPM
GEOPM_DIR=/mnt/scratch/$USER/geopm-build
wget https://raw.githubusercontent.com/geopm/geopm/refs/heads/dev/geopmdpy/install_user.sh
chmod a+x install_user.sh
./install_user.sh --prefix=$GEOPM_DIR --enable-levelzero
```

All bash blocks should be formatted this way.